src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/profiler.css.twig

```diff
@@ -406,3 +406,21 @@ td.main, td.menu {
 .collapsed-menu #minimizePanelIcon {
     transform: rotate(180deg);
 }
+
+.sf-icon {
+    margin-right: 5px;
+    vertical-align: middle;
+    background-repeat: no-repeat;
+    width: 16px;
+    height: 16px;
+}
+.sf-icon.sf-2x {
```

Comment:
If I'm correct there are three font sizes, so I'd propose to add `medium` and `large` 

@stof  your computation is correct, also results in `32px` here ;)